plugins/modules/cloudfront_distribution.py

```diff
@@ -1711,6 +1738,14 @@ def validate_origin(self, client, existing_config, origin, default_origin_path):
                 origin['custom_headers'] = ansible_list_to_cloudfront_list(origin.get('custom_headers'))
             else:
                 origin['custom_headers'] = ansible_list_to_cloudfront_list()
+            if 'origin_shield' in origin:
+                origin_shield = origin.get('origin_shield')
+                if origin_shield.get('enabled') is not None:
+                    if origin_shield['enabled']:
+                        origin_shield_region = origin_shield.get('origin_shield_region').lower()
```

Comment:
You will get this exception if `origin_shield_region` is not set and default on the `None` type:

```
>>> None.lower()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'lower'
```

You should do the `lower()` call later when you're sure you've got a String.